src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -455,6 +455,12 @@
 			"true",
 			"# If Towny should show players the townboard when they login"
 	),
+	GTOWN_SETTINGS_OUTSIDERS_PREVENT_PVP_TOGGLE(
+			"global_town_settings.outsiders_prevent_pvp_toggle",
+			"true",

```

Comment:
Change to false, we want this feature off by default.

Reply:
Oversaw this, yeah ok
 all good